docker-compose.yml

```diff
@@ -164,6 +164,7 @@ services:
     volumes:
       - ./msar/static:/code/static
       - ./msar/media:/code/media
+      - ./sso.yml:/code/sso.yml
```

Comment:
@mathemancer I tried running docker compose without the `sso.yml` file and docker creates a new directory named `sso.yml/` if `sso.yml` is not present. This is not a breaking change but maybe we should consider keeping `sso.yml` within the image instead of the `.example` varient, that way:
- The newest `sso.yml` will be copied to users system on first run with the new volume changes.
- If the docker compose file and the sso.yml stays at the same location any changes made to sso.yml by us won't affect upgrades, since sso.yml will already exist and mount onto the container.